README.md

```diff
@@ -112,14 +111,27 @@ with `DataIntegrityProof` proof type using the `ecdsa-rdfc-2019`,
 To add your implementation to this test suite, you will need to add 2 endpoints
 to your implementation manifest.
 - A credential issuer endpoint (`/credentials/issue`) in the `issuers` property.
+  - An optional id property specifying the issuer can be set alongside the endpoint.
+  - If provided, the specified issuer id will be used as the issuer property on verifiable credentials
+    in the tests.
+  - The issuer id should be easy for other implementers to dereference.
+    - It is recommended the issuer id be `did:key`.
 - A credential verifier endpoint (`/credentials/verify`) in the `verifiers`
 property.
+- An optional `vcHolder` endpoint can be added for `ecdsa-sd-2023` selective disclosure tests.
+  - The vcHolder endpoint route is meant to be `/credentials/derive`
+  - The endpoint is expected to accept a JSON object with 2 properties:
+    - `options` which is a JSON object.
+    - `options.selectivePointers` which is an array of strings.
+    - `verifiableCredential` which is a previously signed JSON verifiable credential.
+    - The endpoint needs to use `ecdsa-sd-2023` for the derived verifiable credential.
 
 All endpoints will require a cryptosuite tag of `ecdsa-rdfc-2019`,
 `ecdsa-jcs-2019`, and/or `ecdsa-sd-2023`. Alongside this cryptosuite tag, you
-must also specify the `supportedEcdsaKeyTypes` property, parallel to `tags`
+will need to specify the `supportedEcdsaKeyTypes` property, parallel to `tags`
 listing the ECDSA key types issuable or verifiable by your implementation.
 Currently, the test suite supports `P-256` and `P-384` ECDSA key types.
+A `vcHolder` tag is required for the `vcHolder` endpoints.
```

Comment:
Can we avoid this tag if they've supplied the additional options data and tagged the endpoint with `ecdsa-sd-2023`?